### ISSUE TITLE
Any reason to delete 'sigma'  in smooth_L1_loss_layer?

### ISSUE THREAD

**Issue by jianan86** [weiliu89/caffe#639]
Hi,
I find that the smooth L1 loss layer in SSD is a little different from faster-rcnn .What motivated you to delete 'sigma' in smooth L1 loss layer? Is the parameter 'sigma'  only suitable for RPN?
Really appreciate if anyone could give me some references or explanations on it.

**Comment by wk910930** [comment]
I guess the `sigma` value doesn't change much to the final results.

**Comment by gauenk** [comment]
Yeah but it seems odd to go out of his way to modify the code and eliminate sigma... If he doesn't want it, just set it to one... @weiliu89 Do you mind shedding some light on this?
